test/test_location.js

```diff
@@ -315,7 +324,7 @@ describe('states.location', function() {
             .check(function(api) {
                 var contact = api.contacts.store[0];
                 assert.equal(contact.extra[
-                    'location:geometry:bounds:northeast:lng'], 
+                    'location:geometry:bounds:northeast:lng'],
                     '18.4575469');
             })
             .run();
```

Comment:
With `store_fields` gone, I'm wondering if this test is still necessary. If we do keep it, I think the test description needs to change.